cmd/vvgo/main.go

```diff
@@ -31,6 +31,10 @@ func NewDefaultConfig() Config {
 			MaxContentLength: 1e6,
 			BasicAuthUser:    "admin",
 			BasicAuthPass:    "admin",
+			SheetsBucketName: "sheets",
+			ClixBucketName:   "clix",
+			PartsBucketName:  "parts",
+			PartsLockerName:  "parts.lock",
```

Comment:
```suggestion
			PartsLockerKey:  "parts.lock",
```
Key instead of Name emphasizes that it's a Redis Key.